client/extensions/woocommerce/app/stats/index.js

```diff
@@ -2,19 +2,40 @@
  * External dependencies
  */
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
 import Main from 'components/main';
 import StatsNavigation from './stats-navigation';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import StatsChart from './stats-chart';
 
-export default class Stats extends Component {
+class Stats extends Component {
```

Comment:
This is a good point. Adding to this, due to eslint's jsx classname rules, there may be some conflicts in CSS namespaces with my current naming structure `stats-`. I hope to take a better look at naming in a separate PR.